app/routes/crypto.py

```
@@ -42,115 +36,74 @@ class SwapRequest(BaseModel):
-        return wallet
-    wallet = Wallet(user_id=user_id, currency=currency, purpose=purpose, storage_type="cold" if purpose == "reserve" else "hot", balance=0.0)
-    db.add(wallet)
-    db.commit()
-    db.refresh(wallet)
-    return wallet
-
 @router.post("/send")
 def send_crypto(
     req: CryptoSendRequest,
-    request: Request,
     current_user: User = Depends(get_current_user),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)